pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about-xrefs-validation-in-pantheon.adoc

```diff
@@ -0,0 +1,46 @@
+[id="about-xrefs-validation-in-pantheon_{context}"]
+= About xref validation in Pantheon
+
+[role="_abstract"]
+When you upload a Git repository to {ProductShortName}, each module and assembly in the repository is built as an html document. During this build, {ProductShortName} checks whether xrefs in each module and assembly are valid.
+
+If {ProductShortName} finds invalid xrefs in a module or assembly, a red alert icon is displayed alongside it in the Search Page.
+
+Only the following forms of xref are validated:
```

Comment:
@theashiot Please state that although according to the Asciidoc standard/spec/rules/whatever you can use `xref:<id>[Title]`, Pv2 doesn't allow it.

Reply:
@stoobie  @theashiot 
Along with this I think it is worth mentioning that xref validation icon might not always indicate that the xref in the adoc file is invalid
eg. document-a.adoc has a xref to document-b.adoc, and during upload the document-a.adoc gets uploaded first, at this point Pantheon thinks that document-b.adoc does not exist, hence, it shows a xref validation. However, when user rerenders the HTML for this document (Using the Generate draft HTML on the document details page) or publishes this document, the xref validation gets resolved and the icon is removed

Reply:
@noopurmath3 Just to make sure I understand you, in the scenario you mention, Pantheon is only able to validate the xref after the user actively does something to render the HTML? It seems like it should validate it as soon as documente-b.adoc is uploaded, even before rendering the HTML.

Reply:
The xref validation is triggered every time the asciidoc source is being converted to html. The following situation qualifies for when xref validation is executed:

1. import a repository from git to pantheon2
2. regenerate draft HTML
3. publish a module/assembly